Block/Analytics.php

```diff
@@ -76,4 +78,18 @@ protected function isRejected($action)
         ];
         return in_array($action, $rejected);
     }
+    
+    public function getEvents() {
+        $cartEvents    = '';
+        $pageEvent     = $this->getEvent();
+        $sessionEvents = $this->sessionEvents->getSessionEvents($this->helper::ADD_TO_CART);
```

Comment:
WTF? we discussed that the key should be removed?!

Reply:
I have no idea what are you showing me here :/

Reply:
I think I'm lost on this one. Do you remember why we agreed to remove that param ($type) in a first place ?